### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Jaws/NVDA][MAS4.2.1][IntelliJ][Wipro]:Jaws and NVDA are not announcing the name of the combo boxes in "Push Image" dialog

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1468]
**Repro Steps:**

1. Open and Start IntelliJ.
2. Create a new java project.
3. Tab to and select Project ->Azure->Push Image.
4. In "Push Image" dialog ,  tab to container registry and Artifact combo boxes and observe the behaviour of Jaws and NVDA

**Actual Result:**
Jaws and NVDA is not announcing the name of the  combo boxes

**Jaws behaviour:**
combo box closed
blank
1 of 1
To change the selection use arrow keys
combo box closed
blank
1 of 1
To change the selection use arrow keys

**NVDA Behaviour:** 
combo box  collapsed
combo box  collapsed
combo box  collapsed

**Expected Result:**
Jaws and NVDA should announce the name of the combo boxes
Name should be 
Container Registry
Artifact

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
OSBuild Version: Version 1709(Os Build 16299.192)
![push image-combo box-explorer](https://user-images.githubusercontent.com/22956130/36102603-d3c19cb0-1032-11e8-9895-cd2c1001b3a5.jpg)
![push image-combobox-name-empty](https://user-images.githubusercontent.com/22956130/36102608-d8c38142-1032-11e8-862c-919b5e5b9b97.jpg)
![ca0374c1](https://user-images.githubusercontent.com/22956130/36102615-db20a320-1032-11e8-8a83-d4182f10b21a.jpg)

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749924
